### DISCUSSION TITLE
事件文档里的一些错误

### DISCUSSION THREAD

**Discussion by Saplonily** [Mrs4s/go-cqhttp#1692] [General]
在go-cqhttp的文档里 [链接](https://docs.go-cqhttp.org/event/#%E6%B6%88%E6%81%AF%E4%B8%8A%E6%8A%A5)  
`请求上报` 和 `通知上报` 标题下面的文字都是  
>post_type 为 request 的上报会有以下有效通用数据  

`通知上报` 下的文字是否应改为下面这样?

>post_type 为 **notice** 的上报会有以下有效通用数据  

刚入门，被这里迷惑了好久(((

**Comment by Akegarasu** [comment]
是，应该是文档打错了
